pkg/storage/allocator_test.go

```diff
@@ -659,6 +667,7 @@ func TestAllocatorRebalance(t *testing.T) {
 			config.Constraints{},
 			testRangeInfo([]roachpb.ReplicaDescriptor{{StoreID: 3}}, firstRange),
 			storeFilterThrottled,
+			true,
```

Comment:
We should just be able to pass `false` into all of these test call sites, since the `Settings` objects are already configured properly.

Reply:
Done.